inc/saved-links/display-recent.php

```diff
@@ -3,14 +3,14 @@
   * @package Link_Roundups
   * @since 0.1
   */
-  
+
 /*
-* Recent Saved Links Custom Meta Panel 
+* Recent Saved Links Custom Meta Panel
 * for Link Roundups Post Type
 */
 
 /** WordPress Admin Bootstrap */
-require_once('../../../wp-admin/admin.php');
+require_once('../../../../../wp-admin/admin.php');
```

Comment:
It is nutty, but this file is accessed directly so it doesn't know about `ABSPATH` or `WP_CONTENT_DIR`. We should probably change the way this works to use a WP rewrite or some sort of blessed query var.